### DISCUSSION TITLE
Freeform 5 No longer able to place the reCAPTCHA or Submit button in specific location

### DISCUSSION THREAD

**Discussion by davidcraigmyle** [solspace/craft-freeform#1153] [Questions & Answers] [answered]
**Describe the bug or issue you're experiencing**
Freeform 4 let you drag and drop the reCAPTCHA where you need it in the form.
More often than not it will be right above the submit button, but with some styles, it'd help to be able to put it in line or to order it. 
This is the same with the buttons, these are now forced to the bottom and can't be put in line with a reCAPTCHA.
Is there a reason this was removed?

**Expected behavior**
reCAPTCHA and button to be a drag and drop element like in Freeform 4 to allow reordering of fields.

**Craft & Plugin Info (please complete the following information):**
- Craft Version: 4.8.2
- Freeform Version: 5.1.4
- Freeform Edition: Lite
- Fresh Install or Upgrade: Upgrade

**Comment by kjmartens** [comment, marked as answer]
Hi @davidcraigmyle,

The reasoning behind this was that it would theoretically make form-building smoother for most customers if we inserted it automatically. It's also something that only applies to reCAPTCHA v2 Checkbox, which is now considered the least effective reCAPTCHA option available, as more and more sites are using reCAPTCHA v3. 🙂 

That said, I understand that the `Lite` edition currently just includes the `v2 Checkbox` version.

Something you can do in Freeform 5.x is [apply custom classes and attributes to the reCAPTCHA wrapper div](https://docs.solspace.com/craft/freeform/v5/templates/formatting/#on-captcha-wrapper). This could allow you to at least use CSS to wrangle the positioning:

``` twig
{{ form.render({
    captchas: {
        class: ["custom-class", "another-class"],
        "data-test": true,
    }
}) }}
```

Additionally, we do offer manual implementation options for Submit buttons. Manually placing or adding classes/attributes to those may also aid with getting your Submit buttons and reCAPTCHA field to be what you need. See the guide for Submit buttons here:
https://docs.solspace.com/craft/freeform/v5/templates/formatting/#submit-button-customization
